src/mmw/js/src/core/models.js

```diff
@@ -57,9 +57,11 @@ var MapModel = Backbone.Model.extend({
     },
 
     setNoHeaderSidebarSize: function(fit, sidebarWidth) {
-        var hasSidebar = true,
-            hasProjectHeader = false;
-        this._setSizeOptions(fit, hasProjectHeader, hasSidebar, sidebarWidth);
+        this._setSizeOptions({
+            fit: fit,
```

Comment:
Just to clarify: `fit` & `sidebarWidth` must dynamic based on the screen size? If that's incorrect, we could consider just setting the default values here.

Reply:
This could probably be set up with smarter defaults, but going to defer that for a more proper refactor. `fit` is a bool that determines if the map should fit to the aoi after resize, and `sidebarWidth` is a mode ... `""` or `"wide"`